bottlecap/src/traces/trace_processor.rs

```diff
@@ -360,7 +361,7 @@ impl TraceProcessor for ServerlessTraceProcessor {
             test_token: None,
         };
 
-        let builder = SendDataBuilder::new(body_size, payload, header_tags, &endpoint)
+        let builder = SendDataBuilder::new(body_size, payload.clone(), header_tags, &endpoint)
```

Comment:
I have to clone the processed trace payload so I can return it to generate stats. Let me know if you have a more efficient approach.